recipe/meta.yaml

```diff
@@ -9,18 +9,18 @@ package:
 
 build:
   number: {{ build_number }}
-  string: {{ build_number }}_cpython
+  string: {{ build_number }}_{{ py_impl }}
   skip: True  # [win]
 {% if (cross_target_platform == "osx-arm64" and not (version == "3.8" or version == "3.9")) or (cross_target_platform == target_platform) %}
   skip: True
 {% endif %}
   run_exports:
     strong:
-      - python {{ version }}.*
+      - python {{ version }}.* *_{{ py_impl }}
```

Comment:
@isuruf Is there a reason the implemention wasn't included here before? I'm not sure if it's needed but it feels odd to omit it

Reply:
Can't remember. I was thinking we could use cpython as build and pypy as host, but that might not be viable.